docs/attachconsole.md

```diff
@@ -5,6 +5,9 @@ author: bitcrazed
 ms.author: richturn
 ms.topic: article
 keywords: console, character mode applications, command line applications, terminal applications, console api
+f1_keywords:
+- WINCON/AttachConsole
```

Comment:
So this one is the same story as `AllocConsole` but is shown as in the other header. It was in `wincon.h` and now it's in `consoleapi.h`. Should both be listed?